0015-update_rfc_process/README.md

```diff
@@ -0,0 +1,125 @@
+---
+RFC: 0015
+Author: Roni Choudhury
+Created: 2021-04-26
+Last-Modified: 2021-04-26
+Supersedes: 0003
+---
+
+# RFC 0015: Adopt Python-esque process for creating RFCs
+
+This RFC describes a process for creating RFCs that adapts the Python/Django
+methodologies for creating "PEPs" and "DEPs" (Python/Django Enhancement
+Proposals). [Django's
+process](https://github.com/django/deps/blob/master/final/0001-dep-process.rst)
+is detailed via its own DEP.
+
+## Motivation
+
+Our previous process for creating RFCs is tedious and error prone, and ends up
+hiding RFC documents in various directories which tend to obscure the history of
+the project. One of the goals of writing RFCs is to provide a written history of
+development for retrospectives and new developers alike. Another goal is for
+people to easily be able to write RFCs (whether in this project, where
+willingness to do so is high, or in other projects, where a similar process may
+not have as much buy in).
+
+To these ends, we're simplifying the process laid out in 0003 to eliminate some
+of the steps involved in writing an RFC. In the future, tooling may help to
+support this streamlined process, leaving little or no friction to writing RFCs.
+
+## Proposals
+
+### New Process for RFC Creation/Review/Acceptance/Rejection
+
+1. The author clones this repository, creates a new directory in the top level
+   directory with a prospective RFC number based on the highest numbered RFC
+   existing so far `0047-new_proposal/`, if `0046-existing_proposal/` is the
+   highest-numbered proposal there), and copies the root-level `template.md`
+   file there as `README.md`.
+2. The author modifies the new `README.md` file to contain their newly proposed
+   RFC, taking care to fill in the metadata information in the header block. If
+   the new RFC is intended to supersede an older one, its metadata should
+   reflect that (see below for details).
+3. If supplemental materials are needed (this should be generally rare), they
+   can go into the new directory alongside the `README.md` file.
+4. The author opens a pull request. The title of the pull request should be the
+   RFC title preceded by a prefix of `RFC:`. A PR description is optional, and
+   can generally be omitted if the PR is simply introducing a new RFC (since the
+   change set itself will detail the reason for opening the PR).
+5. Reviewers provide feedback, engage in discussion, etc., just as in a normal
+   GitHub-based code review.
+6. The RFC becomes "settled". Settlement involves one of several paths of
+   action, detailed next.
+
+### Settling an RFC
+
+RFCs can be settled in several ways.
+
+**Acceptance.** An RFC can be accepted as written (which includes mutually
+agreed-upon edits).
+
+1. Two reviewers provide a review approval.
+2. The author changes the `Status` metadata item to `accepted`.
```

Comment:
Maybe this should be the default since this is the most likely path?

Reply:
See 8b0f8435769a4b49d3fbcdda408bfed19e5507de.